paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetActivityTest.kt

```diff
@@ -268,16 +268,22 @@ internal class PaymentSheetActivityTest {
         scenario.launch(intent).onActivity { activity ->
             // wait for bottom sheet to animate in
             idleLooper()
-            assertThat(activity.viewBinding.message.isVisible).isFalse()
 
-            viewModel.onError("some error")
-            assertThat(activity.viewBinding.message.isVisible).isTrue()
-            assertThat(activity.viewBinding.message.text.toString()).isEqualTo("some error")
+            runTest {
```

Comment:
I don’t think we need this here. We’re testing that the `viewBinding` is updated as expected; testing `viewModel.selection` should be done elsewhere.